apps/np-atobarai/src/app/api/webhooks/saleor/base-use-case.ts

```diff
@@ -0,0 +1,53 @@
+import { SaleorApiUrl } from "@saleor/apps-domain/saleor-api-url";
+import { Logger } from "@saleor/apps-logger";
+import { BaseError } from "@saleor/errors";
+import { err, ok } from "neverthrow";
+
+import { AppConfigRepo } from "@/modules/app-config/repo/app-config-repo";
+
+import { AppIsNotConfiguredResponse } from "./saleor-webhook-responses";
+
+export abstract class BaseUseCase {
+  private appConfigRepo: Pick<AppConfigRepo, "getChannelConfig">;
```

Comment:
appConfigRepo can be abstract too, no need to passing it here when it's abastract